examples/codex-workflow/Makefile

```diff
@@ -0,0 +1,45 @@
+SOLUTION_NAME := ${USER}codexworkflow
+
+# Define a default goal.
+default: test
+
+
+clean:
+	@test -d ./"${SOLUTION_NAME}" && rm -r ./"${SOLUTION_NAME}" || echo "nothing to clean"
+
+# Validate target to check solution files are correct.
+fork: clean
+	@echo "forking solution..."
+	fsoc solution fork --source-dir=package ${SOLUTION_NAME} --verbose
+	@echo "Solution forked."
+
+
+# Validate target to check solution files are correct.
+.PHONY: validate
+validate: fork
+	@echo "Validating solution..."
+	fsoc solution validate --directory ${SOLUTION_NAME} --tag=base -v
+	@echo "Solution validated."
+
+# Push target to deploy your solution.
+push:
+	@echo "Pushing solution..."
+	fsoc solution push --directory ${SOLUTION_NAME} --tag=base
+	@echo "Solution pushed."
+
+# Test target that needs to run fork and local validation
+test:
```

Comment:
shouldnt this call validate? From top level we do `lint` and `validate` calls. markdown-lint automatically happens on all md files as part of the top level job. We can remove that.

Reply:
We can solve these issues in a separate PR, as there is nothing to validate at this point.